pkg/tracker/disputeChecker_test.go

```diff
@@ -65,8 +71,9 @@ func TestDisputeCheckerOutOfRange(t *testing.T) {
 }
 
 func execEthUsdPsrs(ctx context.Context, t *testing.T, psrs []*IndexTracker) {
+	logger := util.SetupLogger("debug")
 	for _, psr := range psrs {
-		err := psr.Exec(ctx)
+		err := psr.Exec(ctx, logger)
```

Comment:
Yeah here we also need to pass the logger to the constructor(so that we can have the component name in the logs) and not the actual struct method.